api/sse/tasks.py

```diff
@@ -27,6 +39,37 @@ def send_environment_update_message(environment_key: str, updated_at):
     response.raise_for_status()
 
 
+if settings.AWS_SSE_LOGS_BUCKET_NAME:
+
+    @register_recurring_task(
+        run_every=timedelta(seconds=60),
+    )
+    def update_sse_usage():
+        with influxdb_client.write_api(
+            write_options=WriteOptions(batch_size=1000, flush_interval=2000)
+        ) as write_api:
+            for log in sse_service.stream_access_logs():
+                environment = Environment.get_from_cache(log.api_key)
+
+                if not environment:
+                    logger.warning("Invalid  api_key %s", log.api_key)
+                    continue
+
+                record = _get_influx_point(environment, log.generated_at)
+                write_api.write(bucket=settings.SSE_INFLUXDB_BUCKET, record=record)
+
+
+def _get_influx_point(environment: Environment, event_time: str) -> Point:
+    return (
+        Point("sse_call")
+        .field("request_count", 1)
```

Comment:
I think we can aggregate here? Why send a request to influx for each data point?

Reply:
Yeah, good shout(I did not do it initially because we loose granularity, but I guess it makes sense)

Reply:
Yeah, interesting - I guess we've lost the granularity on the timestamp. I'm keen to understand though, how granular the log files are in the first place?

Reply:
The file is simple access log, i.e: one line per request

Reply:
But how often do the log files roll over? Isn't that therefore indicative of how granular our data will be?

Reply:
That can be configured in fastly...